fix: Show "1 point" instead of "1 points"

surajshetty3416 opened the pull request:
Before:
<img width="252" alt="Screenshot 2019-05-15 at 11 42 27 AM" src="https://user-images.githubusercontent.com/13928957/57752851-1bb80800-7708-11e9-80b2-c1b1403fbe9c.png">

After:
<img width="250" alt="Screenshot 2019-05-15 at 11 47 50 AM" src="https://user-images.githubusercontent.com/13928957/57752893-35594f80-7708-11e9-9480-aa99c9935ca5.png">

surajshetty3416 committed:
fix: Show "1 point" instead of "1 points"

rmehta commented on frappe/social/doctype/energy_point_log/energy_point_log.py:
```diff
@@ -64,6 +64,13 @@ def get_alert_dict(doc):
 
 	return alert_dict
 
+def get_points_text(points):
```
not translatable, you have to write the whole text in _('....')

surajshetty3416 committed:
fix: Make text translatable

Alchez commented on frappe/social/doctype/energy_point_log/energy_point_log.py:
```diff
@@ -37,20 +37,20 @@ def get_alert_dict(doc):
 	owner_name = get_fullname(doc.owner)
 	doc_link = frappe.get_desk_link(doc.reference_doctype, doc.reference_name)
-	points = frappe.bold(doc.points)
+	points = doc.points
```
Call the `get_points_text` function once here, and use that everywhere, since you're not changing the variable.

surajshetty3416 committed:
fix: Format string twice to avoid duplicate code

- First formatting to add points text
- Second formatting to add variables aftter translation

netchampfaris commented on frappe/social/doctype/energy_point_log/energy_point_log.py:
```diff
@@ -37,24 +37,29 @@ def get_alert_dict(doc):
+		alert_dict.message=_(message).format(points)
 	elif doc.type == 'Appreciation':
-		alert_dict.message = _('{} appreciated your work on {} with {} points').format(
+		message = '{{}} appreciated your work on {{}} with {{}} {0}'.format(points_text)
```
These strings won't get translated

surajshetty3416 committed:
fix(energy point): Alert messages translation
